pinpoint-zookeeper/docker-compose.yml

```diff
@@ -10,29 +8,45 @@ docker-compose pull && docker-compose upces:
     environment:
       ZOO_MY_ID: 1
       ZOO_SERVERS: server.1=0.0.0.0:2888:3888 server.2=zoo2:2888:3888 server.3=zoo3:2888:3888
+    expose:
```

Comment:
shouldn't these conditions be the same with the conditions in {project dir}/docker-compose file? also applies the same below

Reply:
Okay I'll apply same configuration with it